Project/docker-compose.dev.yaml

```diff
@@ -54,3 +54,15 @@ services:
     healthcheck:
       test: [ "CMD", "redis-cli", "--raw", "incr", "ping" ]
     command: /opt/bitnami/scripts/redis/run.sh --maxmemory 100mb
+
+  frontend:
+    build: 
+      context: ./frontend/
+      dockerfile: Dockerfile.frontend
+    command: npm start
```

Comment:
Jak na koniec dockerfile'a dajesz komendę `npm start` to tutaj już nie musisz. Zostaw tu albo tu.

Reply:
k